wercker.yml

```diff
@@ -1,77 +1,57 @@
-box: wercker/rvm
-# Build definition
+box: drecom/ubuntu-ruby:2.3.1
+
 build:
```

Comment:
https://app.wercker.com/buildstep/5a6ef7bfc16a530100f6b298 says `build/steps is not of required type.`

```
build:
  steps:
```

should this be removed? (I'm not sure whether this fixes the problem...)

Reply:
Thank you for a review!

The pull-req uses wercker classic stack and configure workflow and pipeline on web UI.

e.g. 
![screenshot from 2018-04-02 18-41-21](https://user-images.githubusercontent.com/4487291/38192178-b04f5eda-36a6-11e8-96af-2d40e6b717a8.png)

So, please configure your wercker project `ryotarai/ itamae` :pray: 

see also http://sue445.hatenablog.com/entry/2016/04/07/195627#Wercker%E3%81%AE%E8%A8%AD%E5%AE%9A

Reply:
oh……

Reply:
> The pull-req uses wercker classic stack 

No. This PR already looks like a workflow style.

I think we should change to workflow stack on console.

c.f. https://sue445.hatenablog.com/entry/2016/08/12/133645